qa/sc_fork_one_forced_tx.py

```diff
@@ -124,7 +124,8 @@ def run_test(self):
                                 forced_tx=[low_amount_tx_bytes])
             assert_true(False, "Forced transaction should fail to verify agains stake")
         except Exception as e:
-            assert_true("There was an internal server error." in e.error)
+            logging.info("We had an exception as expected: {}".format(str(e)))
```

Comment:
Assertions throw an exception too, so this try block will either throw because `generate_next_block` throws as expected or because `assert_true(False)` throws. Therefore, this test cannot fail.